exec/java-exec/src/test/java/org/apache/drill/exec/store/parquet/TestParquetFilterPushDown.java

```diff
@@ -388,6 +388,45 @@ public void testTimeStampPredicate() throws Exception {
     testParquetFilterPD(query3, 49, 2, false);
   }
 
+  @Test
+  public void testBooleanPredicate() throws Exception {
+    // Table blnTbl was created by CTAS in drill 1.12.0 and consist of 4 files withe the next data:
```

Comment:
Can we generate files on the fly rather then storing them? This will help to prevent any issues, for example, if files generation will change in the next version.